lib/project-utility/TypeScriptFileUpdate.ts

```diff
@@ -95,9 +95,16 @@ export class TypeScriptFileUpdate {
 					const newObject = ts.createObjectLiteral([routePath, routeComponent, routeData]);
 					this.createdStringLiterals.push(linkPath, linkText);
 
+					const notFoundWildCard = "**";
+					const nodes = ts.visitNodes(array.elements, visitor);
+					const errorRouteNode = nodes.filter(element => element.getText().includes(notFoundWildCard))[0];
+					const resultNodes = nodes
+						.slice(0, nodes.indexOf(errorRouteNode))
+						.concat(newObject)
+						.concat(errorRouteNode);
```

Comment:
@jackofdiamond5 The failing tests turned out to be right - we missed that one. If you encounter a routes array without the "**" wildcard, the `errorRouteNode` will be `undefined` and adding it to the array results in `[node, node, undefined]` which `ts.updateArrayLiteral` won't handle as it expects all valid nodes.